config/initializers/z_rails_5_compatibility.rb

```diff
@@ -12,10 +12,23 @@ def read_association(_preloader, record)
         record.public_send(association_name)
       end
     end
+
+    module PluginMigrations
+      def pending_migrations
+        return true if Foreman.in_setup_db_rake?
+
+        @pending_migrations = ActiveRecord::MigrationContext.new(migrations_paths).needs_migration? if @pending_migrations.nil?
```

Comment:
Can't this return like in the default implementation? Now you get duplicated logs.

Reply:
Right! :dart: done.